ai4good/models/nm/nm_model.py

```diff
@@ -26,10 +28,21 @@ def run(self, p: Parameters) -> ModelResult:
         logging.info("Running network model...")
         p.initialise_age_parameters(graph)
 
-        result = process_graph_bm(p, graph, nodes_per_struct)
-        #result_sq = process_graph_sq(p, graph, nodes_per_struct)
+        # Baseline model, 1 food queue
+        result_baseline = process_graph_bm(p, graph, nodes_per_struct)
+
+        # Baseline model, 1 food queue, WITH interventions
+        result_baseline_interventions = process_graph_bm_interventions(p, graph, nodes_per_struct)
+
+        # Multiple food queuess model: 4, 8
+        result_4_food_queues = process_graph_mq(p, graph, nodes_per_struct, 2)
+        result_8_food_queues = process_graph_mq(p, graph, nodes_per_struct, 4)
+
+        # Multiple food queuess model: 4, 8 WITH interventions
+        result_4_food_queues_interventions = process_graph_mq_interventions(p, graph, nodes_per_struct, 2)
+        result_8_food_queues_interventions = process_graph_mq_interventions(p, graph, nodes_per_struct, 4)
 
         return ModelResult(self.result_id(p), {
             'params': p,
-            'result': result
+            'result': result_baseline
```

Comment:
This will be passed as a generic key 'result' with value depending on the profile selected by user